packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -0,0 +1,339 @@
+                                </Text>
+                            </Collapse>
+                        </Box>
+                    ) : (
+                        <Text fz={12} color="gray.6">
+                            -
+                        </Text>
+                    );
+                },
+            },
+            {
+                id: 'status',
+                label: 'Status',
+                cell: (item) => {
+                    return (
+                        <Stack align="center" justify="center">
+                            {currentLogs(item, logs).length > 0 ? (
+                                <Stack align="center" justify="center">
+                                    {getLogStatusIcon(
+                                        currentLogs(item, logs)[0],
+                                        theme,
+                                    )}
+                                    <Collapse in={opened}>
+                                        <Stack align="center" justify="center">
+                                            {getLogStatusIcon(
+                                                handleScheduledDeliveryLogs(
+                                                    item,
+                                                    logs,
+                                                )[0],
+                                                theme,
+                                            )}
+                                            {getLogStatusIcon(
+                                                sendNotificationLogs(
+                                                    item,
+                                                    logs,
+                                                )[0],
+                                                theme,
+                                            )}
+                                        </Stack>
+                                    </Collapse>
+                                </Stack>
+                            ) : (
+                                <Text fz={12} color="gray.6">
+                                    -
+                                </Text>
+                            )}
+                        </Stack>
+                    );
+                },
+                meta: {
+                    style: { width: '1px' },
+                },
+            },
+        ];
+    }, [users, charts, dashboards, projectUuid, logs, opened, toggle, theme]);
+
+    return (
+        <Table className={classes.root} highlightOnHover>
+            <thead>
+                <tr>
+                    {columns.map((column) => {
+                        return (
+                            <Box
+                                component="th"
+                                key={column.id}
+                                style={column?.meta?.style}
+                            >
+                                {column?.label}
+                            </Box>
```

Comment:
you can directly use <th component if there's nothing special and mantine-y going on with the element